src/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -134,6 +134,9 @@ class Tactic
     // The robot performing this Tactic
     std::optional<Robot> robot;
 
+    // Whether or not this tactic should loop forever by restarting each time it is done
+    bool loop_forever;
```

Comment:
I'm conflicted about making this public. I see the use case you're going for when creating actions to match the tactic (since creating a `loop_forever` action in a non-loop forever tactic doesn't make sense), but I'm not a huge fan of exposing this. Maybe a getter function?

@garethellis0 ?

Reply:
it's gonna be protected, just need it for the subclasses

Reply:
Ah I see. @akhilveeraghanta @garethellis0 @EvanMorcom @LiCody @jordanzliu thoughts?

Reply:
I am generally against subclasses directly accessing member variables of a parent class because it breaks the abstraction that the parent class is trying to provide. I think having a protected getter function and making this variable private would be more appropriate here.

Reply:
I'm generally confused by why we have loopforever in actions *and* tactics. I think I can revert this change and solve it in another issue

Reply:
That's a good point too. We can open discussion in another issue.

Reply:
reverted